src/Intervention/Image/AbstractDecoder.php

```diff
@@ -140,6 +140,10 @@ public function isGdResource()
             return (get_resource_type($this->data) == 'gd');
         }
 
+        if ($this->data instanceOf \GdImage) {
```

Comment:
Looks like the lowercase spelling of `instanceof` is preferred: https://github.com/Intervention/image/search?q=instanceof

Reply:
This was overlooked as well, thank you. Force-pushed now.